addons/calendar/models/mail_activity.py

```diff
@@ -48,3 +49,6 @@ def unlink_w_meeting(self):
         res = self.unlink()
         events.unlink()
         return res
+
+    def _to_store_defaults(self):
+        return super()._to_store_defaults() + [Store.One("calendar_event_id")]
```

Comment:
I don't think calendar event defines any store method, so this will default to just `id`, in which case `Store.One` wrapping is not necessary (and it might actually lead to more data returned than expected if we ever need to add store fields on it).

So either keep `Store.One` and add `[]` as second param to define we only want id, or don't have `Store.One` around the field name.

Reply:
Also missing d.ts for adding this field in JS.